CHANGELOG.md

```diff
@@ -43,6 +44,7 @@ All notable, unreleased changes to this project will be documented in this file.
     - `get_checkout_line_tax_rate`
     - `preprocess_order_creation`
   - additionally, `preprocess_order_creation` was extend with `lines_info` parameter
+- remove `product_updated` webhook in `ProductVariantCreate`, replace `product_updated` with `product_variant_updated` webhook in `ProductVariantChannelListingUpdate` #6963 by @piotrgrundas
```

Comment:
Maybe something like this:
```
Remove triggering a webhook event `PRODUCT_UPDATED`  when calling `ProductVariantCreate` mutation.  Use `PRODUCT_VARIANT_CREATED` instead. 
Remove triggering a webhook event `PRODUCT_UPDATED` when calling  `ProductVariantChannelListingUpdate` mutation. Use `PRODUCT_VARIANT_UPDATED` instead. 
```
@maarcingebala  What do you think about this? You're better than me in formatting msg like this one